packages/netstring/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "@endo/netstring",
+  "version": "0.1.0",
+  "description": "Description forthcoming.",
```

Comment:
is anything likely to prompt us to provide the description? If not, remove it? Or fill it in from the README?